src/ui/component/common/form-components/form-field-price.jsx

```diff
@@ -23,7 +27,7 @@ export class FormFieldPrice extends React.PureComponent<Props> {
 
   handleAmountChange(event: SyntheticInputEvent<*>) {
     const { price, onChange } = this.props;
-    const amount = event.target.value ? parseFloat(event.target.value) : '';
+    const amount = event.target.value ? parseFloat(event.target.value) : undefined;
```

Comment:
`null`, not `undefined`, I think

Reply:
Inputs can't use `null` or react throws errors. Something to do with how javascript handles `null`
https://github.com/facebook/react/issues/11417